src/test/compile-fail/phantom-oibit.rs

```diff
@@ -13,10 +13,12 @@
 // (`impl Trait for ..`))
 
 #![feature(optin_builtin_traits)]
+#![feature(immovable_types)]
 
+use std::marker::Move;
 use std::marker::{PhantomData};
 
-unsafe trait Zen {}
+unsafe trait Zen: ?Move {}
```

Comment:
Did ... you need to add this? In other words, was it injecting new compiler errors otherwise? (If so, what were they?)

If not, then I assume you just added it because its the right way to preserve the previous semantics of the test here?

Reply:
Auto traits cannot have supertraits because that is unsound, so we need to opt-out to `Move`, which is a default bound on `Self` for all traits.